database/seeds/OtherDataSeeder.php

```diff
@@ -0,0 +1,141 @@
+<?php
+
+use Illuminate\Database\Seeder;
+
+use Proto\Models\HashMapItem;
+use Proto\Models\User;
+use Proto\Models\Committee;
+use Proto\Models\Address;
+use Proto\Models\Bank;
+use Proto\Models\Product;
+use Proto\Models\Member;
+use Proto\Models\Activity;
+use Proto\Models\OrderLine;
+use Proto\Models\CommitteeMembership;
+use Proto\Models\ActivityParticipation;
+
+class OtherDataSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+
+        $faker = Faker\Factory::create();
+
+        // List of valid ID's
+        $products = Product::all()->pluck('id')->toArray();
+        $committees = Committee::all()->pluck('id')->toArray();
+
+        // Create users
+        $n = 500;
+        echo "Creating $n users" . PHP_EOL;
+        factory(User::class, $n)->create()->each(function ($user) {
+            /** @var $user User */
+            if (mt_rand(1, 5) > 1) {
+                $user->address()->save(factory(Address::class)->make());
+            }
+            if (mt_rand(1, 5) > 1) {
+                $user->bank()->save(factory(Bank::class)->make());
+            }
+            if (mt_rand(1, 5) > 1) {
+                $user->member()->save(factory(Member::class)->make());
+            }
+        });
+
+        $users = array_merge(array_rand(User::all()->pluck('id')->toArray(), 75), [1]);
+
+        // Create orderlines
+        $n = 10000;
+        echo "Creating $n orderlines" . PHP_EOL;
+
+        $mintime = date('U', strtotime('April 20, 2011'));
```

Comment:
Perhaps it ie enough to create a number of orderlines in the past 3 months or so. Further back does not add extra.

Reply:
Past year it is :)